docs/index.asciidoc

```diff
@@ -1213,6 +1271,97 @@ https://www.elastic.co/blog/elasticsearch-versioning-support[versioning support
 blog] and {ref}/docs-index_.html#_version_types[Version types] in the
 Elasticsearch documentation.
 
+[id="plugins-{type}s-{plugin}-deprecated-options"]
+==== Elasticsearch Output Deprecated Configuration Options
```

Comment:
Deviation from the way we typically handle deprecations in plugins.  (See https://www.elastic.co/guide/en/logstash/current/plugins-inputs-beats.html#plugins-inputs-beats-options.) 

Standards can change over time, but the change needs to be deliberate and communicated clearly.

Reply:
Nevermind. I see where this came from: https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/1118#discussion_r1131892391. 

This is definitely better for new users, but isn't very nice to our existing users.